tests/unit/models/addon-test.js

```diff
@@ -407,6 +407,86 @@ describe('models/addon.js', function() {
       });
     });
 
+    describe('hintingEnabled', function() {
```

Comment:
Can you throw a doc block here describing each permutation you're testing? It's a matrix and it'd be useful to have a picture.

| configuration  | test 1 | test 2 |
| ------------- | ------------- | ---------- |
| hinting  | true  |
| environment  | false  |
| test_command  | set  |
| RESULT | true |